iml-wire-types/wire-types.rs

```diff
@@ -890,7 +890,9 @@ pub enum AlertType {
     StratagemUnconfiguredAlert,
 }
 
-#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, Hash, PartialEq, Eq)]
+#[derive(
+    serde::Serialize, serde::Deserialize, Copy, Clone, Debug, Hash, PartialOrd, Ord, PartialEq, Eq,
+)]
 pub enum AlertSeverity {
     INFO,
     DEBUG,
```

Comment:
I think order is important for Ord to make sense.

Reply:
Good catch.